### ISSUE TITLE
improve sidebar

### ISSUE THREAD

**Issue by Chovin** [OIT-UOG/specify-webportal-installer#23]
let's improve the sidebar design. It looks very old and clunky

Here's [GBIF's](https://www.gbif.org/occurrence/search):
[![image](https://user-images.githubusercontent.com/17536161/54657316-b3d4af00-4b14-11e9-9b7a-014315903808.png)](https://www.gbif.org/occurrence/search)

**Comment by Chovin** [comment]
in #19

**Comment by burdickdr** [comment]
I agree that it would be good if the sidebar and results listing can look more like those on GBIF
